content/brukeropplevelse-og-datadeling/produkter/_index.en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Products
+ingress:
+
+# SEO
+description:
+keywords:
+
+banner__img: /illustrations/illustration-03.png
+
+url: /en/products
```

Comment:
Remember to reflect new structure

Reply:
Applies to all english pages

Reply:
removed all of the English pages for now, until the language design and support are in place again. 🤞